fluidsynth/src/sfloader/fluid_defsfont.c

```diff
@@ -1817,6 +1829,61 @@ fluid_sample_in_rom(fluid_sample_t* sample)
 /*
  * fluid_sample_import_sfont
  */
+#if LIBSNDFILE_SUPPORT
+// virtual file access rountines to allow for handling
+// samples as virtual files in memory
+static sf_count_t sfvio_pos;
+
+static sf_count_t
+sfvio_get_filelen(void* user_data)
+{
+  fluid_sample_t *sample = (fluid_sample_t *)user_data;
+
+  return (sf_count_t)(sample->end + 1 - sample->start);
+};
```

Comment:
Why have you placed these lonesome `;` here and below? Could you remove them? They may cause compiler warnings.